client/trino-client/src/main/java/io/trino/client/spooling/DataAttributes.java

```diff
@@ -88,7 +88,7 @@ public int hashCode()
 
     @JsonCreator
     @DoNotCall
-    public static DataAttributes fromMap(Map<String, Object> attributes)
+    public static DataAttributes fromMap(@JsonProperty("attributes") Map<String, Object> attributes)
```

Comment:
Kida sad that this annotation is now required? I wonder why

Reply:
it was always required but POJO introspection was able to figure it out somehow (which was the source of other issues). Now this is explicit and other places were fine.

Reply:
I mean, if the attribute name is the same as the parameter name, then it could/should be optional? It used to be that the parameter names were not available for reflection, so annotating them was a way to workaround that, but this is no longer true.

Reply:
I think that it works that way for constructors but not for static methods annotated as @JsonCreator